src/debugger/titaniumSourceMapTransformer.ts

```diff
@@ -22,25 +22,27 @@ export class TitaniumSourceMapTransformer extends BaseSourceMapTransformer {
 
 	public async configureOptions (args: TitaniumLaunchRequestArgs) {
 		this.appDirectory = args.projectDir;
-		this.platform = args.platform === 'ios' ? 'iphone' : args.platform;
+		this.platform = args.platform;
 		this.projectType = await determineProjectType(this.appDirectory);
 	}
 
 	public async scriptParsed (pathToGenerated: string, sourceMapUrl: string): Promise<string[]> {
 		if (this.projectType === 'alloy') {
-			const mapDir = path.join(this.appDirectory, 'build', 'map', 'Resources', this.platform);
+			const platform = this.platform === 'ios' ? 'iphone' : this.platform;
+			const mapDir = path.join(this.appDirectory, 'build', 'map', 'Resources', platform);
 			const libDir = path.join(this.appDirectory, 'app', 'lib');
 			const filename = path.basename(pathToGenerated);
 			const relative = path.relative(path.join(this.appDirectory, 'app'), pathToGenerated);
-			const dir = path.dirname(relative);
 			const isLib = pathToGenerated.includes(libDir);
+			let dir = path.dirname(relative);
```

Comment:
It looks like we only use `dir` in the else so maybe move the declaration there?